src/components/frontend-client-base.tsx

```diff
@@ -40,13 +44,11 @@ export function FrontendClientBase({
     if (typeof window === 'undefined') {
       // load instance data for server side rendering
       // Note: using require to avoid webpack bundling it
+      // eslint-disable-next-line
```

Comment:
Why? You should at least be specific here and only disable the rules that aren't useful for this special case.

Reply:
I thought that might be a bit verbose… :) 
```
// eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```

Reply:
Hm. Well, that's an annoying case then. I'd still prefer the verbose variant to document which rules fail here (so that e.g. a newly introduced rule still triggers here possibly and the next one that touches the code can possibly review that).

Reply:
Check.